src/app/_helpers/calcprogress.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { WorkflowExecution } from '../_models';
+
+import { calcProgress } from './calcprogress';
+
+function makeWorkflowExecution(currIndex: number, progressStatuses: (number)[]): WorkflowExecution {
```

Comment:
you can just use `number[]` instead of `(number)[]`

Reply:
Done